src/store/CategoricalDisplayStore.js

```diff
@@ -268,6 +277,7 @@ export default class CategoricalDisplayStore {
         };
 
         for (const category of filteredItems) {
+            category.active = true;
```

Comment:
on Search, we should open all position options, as a result, we open all categories